release-notes.rst

```diff
@@ -1,6 +1,12 @@
 Relese notes for Chevah KeyCert
 ###############################
 
+1.4.2 - 06/01/2017
```

Comment:
this is fine for now... but this should be a 1.5.0 release, as this is not a bugfix release

you should get familiar with semantic versioning  http://semver.org/ :)